motoko/invoice-canister/install-local.sh

```diff
@@ -3,7 +3,7 @@
 set -e
 
 dfx stop
-dfx start --background --clean
+dfx start --background --clean --host 127.0.0.1:8000
```

Comment:
Does this need to be forced to port 8000?

Reply:
the e2e tests are hard-coded in multiple places to port 8000. There would probably be a way to move this somewhere else, but I didn't have the time to look into that

Reply:
was able to remove it 😄